lib/checks/aria/no-implicit-explicit-label.js

```diff
@@ -0,0 +1,21 @@
+const { aria, text } = axe.commons;
+
+const role = aria.getRole(node, { noImplicit: true });
+this.data(role);
+
+const labelText = text.sanitize(text.labelText(virtualNode)).toLowerCase();
+const accText = text.sanitize(text.accessibleText(node)).toLowerCase();
+
+if (!accText && !labelText) {
+	return undefined;
```

Comment:
I think this needs to return false. If there is no label, this check has nothing to report.